cartography/models/spacelift/run.py

```diff
@@ -0,0 +1,185 @@
+    stack_id: PropertyRef = PropertyRef("stack_id")
+    triggered_by_user_id: PropertyRef = PropertyRef("triggered_by_user_id")
+    account_id: PropertyRef = PropertyRef("account_id")
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToAccountRelProperties(CartographyRelProperties):
+    """
+    Properties for the RESOURCE relationship between a Run and its Account.
+    """
+
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToAccountRel(CartographyRelSchema):
+    """
+    RESOURCE relationship from a Run to its Account.
+    (:SpaceliftRun)<-[:RESOURCE]-(:SpaceliftAccount)
+    """
+
+    target_node_label: str = "SpaceliftAccount"
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {"id": PropertyRef("account_id", set_in_kwargs=True)},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "RESOURCE"
+    properties: SpaceliftRunToAccountRelProperties = (
+        SpaceliftRunToAccountRelProperties()
+    )
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToStackRelProperties(CartographyRelProperties):
+    """
+    Properties for the GENERATES relationship between a Stack and its Run.
+    """
+
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToStackRel(CartographyRelSchema):
+    """
+    GENERATES relationship from a Run to its parent Stack.
+    (:SpaceliftRun)<-[:GENERATES]-(:SpaceliftStack)
+    """
+
+    target_node_label: str = "SpaceliftStack"
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {"id": PropertyRef("stack_id")},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "GENERATES"
+    properties: SpaceliftRunToStackRelProperties = SpaceliftRunToStackRelProperties()
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToUserRelProperties(CartographyRelProperties):
+    """
+    Properties for the TRIGGERED relationship between a User and a Run.
+    """
+
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToUserRel(CartographyRelSchema):
+    """
+    TRIGGERED relationship from a Run to the User who triggered it.
+    (:SpaceliftRun)<-[:TRIGGERED]-(:SpaceliftUser)
+    """
+
+    target_node_label: str = "SpaceliftUser"
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {"id": PropertyRef("triggered_by_user_id")},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "TRIGGERED"
+    properties: SpaceliftRunToUserRelProperties = SpaceliftRunToUserRelProperties()
+
+
+@dataclass(frozen=True)
+class SpaceliftRunToWorkerRelProperties(CartographyRelProperties):
+    """
+    Properties for the EXECUTES relationship between a Worker and a Run.
```

Comment:
Similar to GENERATED and TRIGGERED I do think that instead of EXECUTES and AFFECTS it's probably more correct to say EXECUTED and AFFECTED because this is all in the past and runs are all one-off